src/wazuh_modules/inventory_harvester/src/wcsModel/inventoryHardwareHarvester.hpp

```diff
@@ -0,0 +1,28 @@
+/*
+ * Wazuh inventory harvester
+ * Copyright (C) 2015, Wazuh Inc.
+ * March 20, 2025.
+ *
+ * This program is free software; you can redistribute it
+ * and/or modify it under the terms of the GNU General Public
+ * License (version 2) as published by the FSF - Free Software
+ * Foundation.
+ */
+
+#ifndef _INVENTORY_HARDWARE_HARVESTER_HPP
+#define _INVENTORY_HARDWARE_HARVESTER_HPP
+
+#include "reflectiveJson.hpp"
+#include "wcsClasses/agent.hpp"
+#include "wcsClasses/hardware.hpp"
+
+struct InventoryHardwareHarvester final
+{
+    Agent agent;
+    Hardware host;
+
+    REFLECTABLE(MAKE_FIELD("host", &InventoryHardwareHarvester::host),
```

Comment:
This structure isn't the same as the one requested, because here `observer` is inside `host`; but it isn't like that in the example JSON of the issue.

Reply:
I changed and fixed